test/commands_map.ts

```diff
@@ -41,58 +41,51 @@ var commands_map: CommandValuesMap = {
         server: function (res, sendActivity) {
             sendActivity(res, server_content.ani_card);
         }
-    },
+    },    
     "carousel": {
         client: function () {
             return document.querySelectorAll('.scroll.next').length > 0;
         },
         server: function (res, sendActivity) {
             sendActivity(res, server_content.car_card);
         }
-    },
+    },    
     "carousel-to-right": {
-        client: async function () {
-
-            async function rightArrowClick(){
-                right_arrow.click();
-                await new Promise((resolve) => {
-                    setTimeout(resolve, 1000);
-                });
-            }
-
+        client: () => new Promise((resolve) => {
             var right_arrow = document.querySelectorAll('.scroll.next')[0] as HTMLButtonElement;
-            
-            // Carousel made of 4 cards.
-            // 2-Clicks are needed to move all carousel to right.
-            // Note: Electron browser width size must not be changed. 
-            await rightArrowClick();
-            await rightArrowClick();
 
-            // Validates if the right_arrow button is disabled.             
-            return right_arrow.getAttribute('disabled') != null;
-        },
+            // Carousel made of 4 cards.
+            // 3-Clicks are needed to move all carousel to right.
+            // Note: Electron browser width size must not be changed.             
+            right_arrow.click();
+            setTimeout(() => {
+                right_arrow.click();
+                setTimeout(() => {
+                    right_arrow.click();
+                    setTimeout(() => {
+                        resolve(right_arrow.getAttribute('disabled') != null);
+                    }, 2000);
+                }, 500);
+            }, 500);
+        }),
```

Comment:
This is unrelated for markdown-it. Some tests were affected due the change of the library and the using of async / await words.